test/test.ml

```diff
@@ -37,6 +37,7 @@ let parse_test_case_json s =
   | `Assoc [("test_cases", `List seq)] -> List.map seq ~f:(function
     | `Assoc assoc -> List.Assoc.map assoc ~f:(function 
       | `String v -> v
+      | `Null -> ""
```

Comment:
Some of the `patch` versions in the `yml` file are empty. I don't know whether this was always the case (and the newer `yaml` library interprets these differently now), or whether such entries didn't exist before.